### PR TITLE
Restore `openfl-tutorials` as installable package

### PR DESCRIPTION
**Pull request by MasterSkepticista**
`openfl-tutorials` were excluded as packages since #1187. This caused an issue installing requirements for experimental API activation.

Edit: I still think tutorials need not be included as "installable package" and needs to be revisited.

### CODE REVIEW

File: openfl-workspace/experimental/workflow/requirements.txt [hunk 1]
```diff
@@ -1,10 +1,6 @@
-astor==0.8.1
-chardet
-charset-normalizer
 dill==0.3.6
-matplotlib>=2.0.0
 metaflow==2.7.15
 nbdev==2.3.12
 ray==2.9.2
-torch
-torchvision
```
Comment (MasterSkepticista) [inline openfl-workspace/experimental/workflow/requirements.txt:10]: Removed packages (esp. `torch`, `torchvision`, `matplotlib`) should ideally be installed by the user, as they depend on the example being run.

**Comment by tanwarsh** [comment]
@MasterSkepticista I am facing same issues with these changes.

**Comment by tanwarsh** [comment]
resolving dependencies leads to following error.
```
Error: Invalid value for '--template': 'tf_cnn_histology' is not one of 'template_workspace', '102_aggregator_validation', '101_torch_cnn_mnist', 'vertical_fl_two_party', '301_torch_cnn_mnist_watermarking', '501_pytorch_tinyimagenet_transfer_learning', 'vertical_fl', '104_keras_mnist'.
```

**Comment by MasterSkepticista** [comment]
> @MasterSkepticista I am facing same issues with these changes.

You may have previously enabled experimental interface. Remove `~/.openfl/experimental` file and re-activate using `fx experimental activate`

**Comment by tanwarsh** [comment]
With a fresh virtual environment and installing openfl after that result in the same error.

```
(exp) perfuser@Dev-vm-45:~/shailesh/exp$ fx experimental activate
[20:43:54] WARNING  Requirements file                                                                                                         experimental.py:48
                    /home/perfuser/shailesh/exp/lib/python3.11/site-packages/openfl-tutorials/experimental/workflow/workflow_interface_requir                   
                    ements.txt not found.                                                                                                                       

 ✔️ OK
(exp) perfuser@Dev-vm-45:~/shailesh/exp$ fx
Traceback (most recent call last):
  File "/home/perfuser/shailesh/exp/bin/fx", line 8, in <module>
    sys.exit(entry())
             ^^^^^^^
  File "/home/perfuser/shailesh/exp/lib/python3.11/site-packages/openfl/interface/cli.py", line 336, in entry
    command_group = import_module(module, package)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/perfuser/shailesh/exp/lib/python3.11/site-packages/openfl/experimental/workflow/interface/cli/collaborator.py", line 26, in <module>
    from openfl.experimental.workflow.federated import Plan
  File "/home/perfuser/shailesh/exp/lib/python3.11/site-packages/openfl/experimental/workflow/federated/__init__.py", line 8, in <module>
    from openfl.experimental.workflow.federated.plan import Plan
  File "/home/perfuser/shailesh/exp/lib/python3.11/site-packages/openfl/experimental/workflow/federated/plan/__init__.py", line 8, in <module>
    from openfl.experimental.workflow.federated.plan.plan import Plan
  File "/home/perfuser/shailesh/exp/lib/python3.11/site-packages/openfl/experimental/workflow/federated/plan/plan.py", line 17, in <module>
    from openfl.experimental.workflow.interface.cli.cli_helper import WORKSPACE
  File "/home/perfuser/shailesh/exp/lib/python3.11/site-packages/openfl/experimental/workflow/interface/__init__.py", line 7, in <module>
    from openfl.experimental.workflow.interface.fl_spec import FLSpec
  File "/home/perfuser/shailesh/exp/lib/python3.11/site-packages/openfl/experimental/workflow/interface/fl_spec.py", line 13, in <module>
    from openfl.experimental.workflow.utilities import (
  File "/home/perfuser/shailesh/exp/lib/python3.11/site-packages/openfl/experimental/workflow/utilities/__init__.py", line 12, in <module>
    from openfl.experimental.workflow.utilities.metaflow_utils import MetaflowInterface
  File "/home/perfuser/shailesh/exp/lib/python3.11/site-packages/openfl/experimental/workflow/utilities/metaflow_utils.py", line 19, in <module>
    import ray
ModuleNotFoundError: No module named 'ray'
(exp) perfuser@Dev-vm-45:~/shailesh/exp$ 
```

Even after removing this file  ```~/.openfl/experimental```

**Review by tanwarsh — APPROVED** [review]

**Pull request merged**